descriptors/ModuleDescriptor-template.json

```diff
@@ -124,7 +121,7 @@
         {
           "methods": ["GET"],
           "pathPattern": "/bl-users/by-username/{name}/open-transactions",
-          "permissionsRequired": ["users-bl.transactions.get"],
+          "permissionsRequired": ["users-bl.transactions-by-username.get"],
```

Comment:
Permission is not defined in module descriptor

Reply:
Also this permission is not added users-bl.all

Reply:
oh, yep, I forgot to rename new one, it's actually added